scripts/openstack-helm-sealed-secrets.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
So `-e` means we have to catch every error otherwise the script fails. We're not handling the last one so this will make this script start failing.